membership-attribute-service/app/controllers/AttributeController.scala

```diff
@@ -115,13 +115,20 @@ class AttributeController extends Controller with LoggingWithLogstashFields {
     def queryToAccountIds(response: QueryResponse): List[AccountId] =  response.records.map(_.Id)
 
     def getSubscriptions(accountIds: List[AccountId]): Future[Disjunction[String, List[Subscription[AnyPlan]]]] = {
-      def sub(accountId: AccountId): Future[List[Subscription[AnyPlan]]] = {
+      def sub(accountId: AccountId): Future[Disjunction[String, List[Subscription[AnyPlan]]]] =
         subscriptionService.subscriptionsForAccountId[AnyPlan](accountId)(anyPlanReads)
-      }
-      val maybeSubs: Future[List[Subscription[AnyPlan]]] = Future.traverse(accountIds)(id => sub(id)).map(_.flatten)
 
-      maybeSubs map { subs =>
-        if (subs.isEmpty) \/.left(s"Error getting subscriptions for identityId $identityId") else \/.right(subs)
+      val maybeSubs: Future[Disjunction[String, List[Subscription[AnyPlan]]]] = accountIds.traverseU(id => sub(id)).map(_.sequenceU.map(_.flatten))
+
+      maybeSubs.map {
+        _.leftMap { errorMsg =>
+          log.logger.error(s"We tried getting a subscription for a user, but then ${errorMsg}")
+          errorMsg
+        } map { subs =>
+          if(subs.isEmpty)
+            log.logger.info(s"We tried getting subs for user with identityId ${identityId} but there weren't any")
```

Comment:
what's special about the empty case? why not log the number of subs returned (zero or more..)